app/dashboards/api_consumer_dashboard.rb

```diff
@@ -0,0 +1,71 @@
+require 'administrate/base_dashboard'
+
+class ApiConsumerDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    id: Field::Number,
+    public_api_key: Field::String,
+    private_api_key: Field::String,
+    contact_email: Field::String,
```

Comment:
One thing I noticed: The web UI enforces that a contact_email must be set, but the database field is not actually `not null`. Wouldn't it be good if the constraints are consistent in the database and in the API, where possible?

In Django, there are two configuration fields in the models: "null" (for the database) and "blank" (for the validation). For fields that should not be empty, I usually set both to `False`.

Reply:
This is inconsistent, correct. But the inconsistency is between Rails model and database, not here.

I made a PR: https://github.com/gfroerli/api/pull/91